src/shared/cluster_utils.c

```diff
@@ -49,11 +49,11 @@ int w_is_worker(void) {
                     free(cl_type);
                 }
             } else if (strcmp(cl_status, "yes")){
-                is_worker = -1;
+                is_worker = 2;
```

Comment:
I think that this value should be `0`, meaning that this node is not a worker (`false`).

Reply:
Should the other case (cluster configuration not found) have also a value of 0? This could also mean that this node is not a worker (or that the user unintentionally erased the cluster config)